### ISSUE TITLE
Output TeX filename has no basename (only extention) when I set Japanese characters to "Project name" on quickstart

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1190]
With 1.2b1, sphinx-quickstart generates conf.py that include no basename files when I set Japanese characters (UTF-8) to "Project name".

```
$ sphinx-quickstart docs
...
> Project name: サンプルドキュメント
> Author name(s): Example
...
> Project version: 1.0
...
```

and i got conf.py including these lines

```
latex_documents = [
  ('index', '.tex', u'サンプルドキュメント Documentation',
   u'Example', 'manual'),
]
```

The second field of latex_documents means output filename of TeX.
So I got file named as ".tex" when i ran "make latex"

```
$ make latex
sphinx-build -b latex -d _build/doctrees   . _build/latex
Making output directory...
Running Sphinx v1.2b1
loading pickled environment... not yet created
building [latex]: all documents
updating environment: 1 added, 0 changed, 0 removed
reading sources... [100&#37;] index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing .tex... index
resolving references...
writing... done
copying TeX support files...
done
build succeeded.

Build finished; the LaTeX files are in _build/latex.
Run `make' in that directory to run these through (pdf)latex (use `make latexpdf' here to do that automatically).
$ find _build/latex
_build/latex
_build/latex/.tex
_build/latex/fncychap.sty
_build/latex/Makefile
_build/latex/python.ist
_build/latex/sphinx.sty
_build/latex/sphinxhowto.cls
_build/latex/sphinxmanual.cls
_build/latex/tabulary.sty
```

It seems man and texinfo have same problem (maybe).

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1190
- Originally reported by: Takeshi Komiya
- Originally created at: 2013-06-08T10:38:02.984

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-06-09 00:59:59+00:00_

Currently, filename accepts 'alphanumeric' and '-', '_' characters.
If project name does not contain these chars, some filenames will be empty.
In this case, I think it is reasonable to the filename `sphinx.tex`.

**Comment by shimizukawa** [comment]
_From Takeshi Komiya on 2013-06-10 08:28:27+00:00_

I can accept quickstart chooses 'sphinx.tex' when basename is empty.
I feel it is good resolution.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-06-13 03:24:32+00:00_

Fix: Output TeX/texinfo/man filename has no basename (only extention) when using multibyte characters to "Project name" on quickstart. Closes #1190

→ <<cset 8c543c11cc58a01b78ff5b6470a87b765e363d69>>
